cmd/eksctl-anywhere/cmd/createcluster.go

```diff
@@ -234,9 +273,7 @@ func (cc *createClusterOptions) directoriesToMount(clusterSpec *cluster.Spec, cl
 }
 
 func buildCliConfig(clusterSpec *cluster.Spec) *config.CliConfig {
-	cliConfig := &config.CliConfig{
-		MaxWaitPerMachine: config.GetMaxWaitPerMachine(),
-	}
+	cliConfig := &config.CliConfig{}
```

Comment:
Looking at this, should those timeout options go in this struct?

Def not blocking: I feel like we probably need a general agreement on how to handle flags/configuration, from reading to passing them down to other entities. That conversation is not for this PR, but we should have it.

Reply:
Agreed we need to have a common way to handle flag config as right now we do not. So I'm completely up to talking it through. 

I decided against using CliConfig because the timeout are only used by ClusterManager. They are not used anywhere else and I did not care for the approach of passing all args to the factory in CliConfig and then having the factory responsible for parsing out the logs to ClusterManager. I thought it would be better to just pass them directly to cluster manager. But again I am open to doing it differently. Whatever the best practice is I want to follow it.

Reply:
Sounds good! Let's take this offline and involve folks who are interested in the topic :)